velox/type/TimestampConversion.h

```diff
@@ -107,6 +107,18 @@ Status daysSinceEpochFromWeekDate(
     int32_t dayOfWeek,
     int64_t& out);
 
+/// Computes the (signed) number of days since unix epoch (1970-01-01).
+/// Returns UserError status if the date is invalid.
+/// @param year Year, can be negative e.g: 1996, -2000
+/// @param month Month of year, e.g: 7
+/// @param weekOfMonth Week of month, 1 ~ 5 e.g: 2
```

Comment:
> 1 ~ 5

This is not very common way to describe a valid range. Perhaps, 

`Week of the month. A value in [1, 5] range. For example, 1 is 1st week, 3 is 3rd week.`

Reply:
Thanks for your advice. I've changed comments.